app/views/course/assessment/answer/programming/_test_cases.html.slim

```diff
@@ -41,11 +41,14 @@ table.table.table-striped.table-hover
             = fa_icon 'check'.freeze
           - elsif test_case_result
             = fa_icon 'times'.freeze
-  - if can?(:grade, answer.answer) && !submission.attempting?
+  - if can?(:grade, answer.answer)
     tbody
       tr
         th colspan=5
           = t('.private')
+          span< class="bg-danger text-danger"
+            span class="glyphicon glyphicon-warning-sign" aria-hidden="true"
```

Comment:
@caramelcat We use font-awesome for icons, can use `=fa 'icon-name'` to display the icon.